requirements.txt

```diff
@@ -5,58 +5,63 @@ blis==0.4.1
 Bottleneck==1.3.1
 catalogue==1.0.0
 certifi==2019.11.28
+cffi==1.13.2
 chardet==3.0.4
 Click==7.0
-cloudpickle==1.2.2
+cloudpickle==1.3.0
 colorama==0.4.3
 coverage==5.0
+cryptography==2.8
 cycler==0.10.0
 cymem==2.0.3
 cytoolz==0.10.1
-dash==1.7.0
-dash-core-components==1.6.0
+dash==1.8.0
+dash-core-components==1.7.0
 dash-html-components==1.0.2
-dash-renderer==1.2.2
-dash-table==4.5.1
-dask==2.9.2
+dash-renderer==1.2.3
+dash-table==4.6.0
+dask==2.10.1
 decorator==4.4.1
 entrypoints==0.3
 fastai==1.0.60
```

Comment:
Have you looked at fastai2 yet? You'll get a much better speedup...

https://github.com/fastai/fastai2

Reply:
So I guess I decided not to use fastai2 just yet because they say they're still in development so I don't want to use it if it's going to be subject to changes later. I thought I would stick with fastaiv1 for now and when fastai2 gets an official release, implement the changes then. I'm not sure exactly how much the code would change but it sounds like they really want to revamp the different classes and how they interact.

Reply:
According to the paper that was released describing fastai2, they're only revamping the lower-level stuff, not the higher-level APIs that you're used to, so the changes should be minimal. Yes, it is in beta, but it's no more or less stable than fastai v1, which has had its own share of issues with backward compatibility and the like. Still worth looking at this as an optimization step, when you get to the point of trying to speed your app up.